src/cam_qhy.cpp

```diff
@@ -291,6 +584,36 @@ bool Camera_QHY::Connect(const wxString& camId)
         return CamConnectFailed(_("Failed to get gain range"));
     }
 
+    if (GetQHYCCDParamMinMaxStep(m_camhandle, CONTROL_OFFSET, &m_offsetMin, &m_offsetMax, &m_offsetStep) == QHYCCD_SUCCESS)
```

Comment:
could we do something like this so if the API call sets m_offsetStep to zero we don't get a divide by zero later on.  Should never happen if the SDK is behaving properly, but prefer not to make that assumption.
```suggestion
    if (GetQHYCCDParamMinMaxStep(m_camhandle, CONTROL_OFFSET, &m_offsetMin, &m_offsetMax, &m_offsetStep) == QHYCCD_SUCCESS && m_offsetStep > 0)
```

Reply:
Hey Andy, I made that change (and now guard against a 0 step in several places) and a few others, as well.